lib/Sema/CSApply.cpp

```diff
@@ -2717,6 +2696,18 @@ namespace {
       llvm_unreachable("Unhandled OverloadChoiceKind in switch.");
     }
 
+    /// Form a type checked expression for the index of a @dynamicMemberLookup
+    /// subscript index parameter.
+    Expr *buildDynamicMemberLookupIndexExpr(StringRef name, SourceLoc loc,
+                                            Type literalTy, DeclContext *dc) {
+      // Build and type check the string literal index value to the specific
```

Comment:
This no longer needs the DeclContext parameter.

Reply:
@CodaFi Good spot, thanks!